### PR TITLE
Multiple pages: Add homepages

### PR DESCRIPTION
**Pull request by principis**
Added some more homepages, see: #2660 
It seemed handy for all the git pages to link directly to the docs although it's not really a homepage. But it is the same website so... Let me know if I need to change anything.

### CODE REVIEW

File: pages/common/gitk.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # gitk
 
 > A graphical git repository browser.
+> Homepage: <https://git-scm.com/docs/gitk>.
+> Homepage: <https://git-scm.com/docs/gitk>.
```
Comment (blunket) [inline pages/common/gitk.md]: You probably didn't mean to dupe this one :slightly_smiling_face:

**Comment by blunket** [comment]
(Sorry for my previous comment -- I screwed up, lol)

**Comment by principis** [comment]
> (Sorry for my previous comment -- I screwed up, lol)

NP :) my script messed it up.

**Comment by waldyrious** [comment]
@sbrl what happened to your insistence on using the slashes at the end of URLs? ;)

In all seriousness, though, I think it might be better if we don't include them because people are more likely to write links without a trailing slash than with them. If we insist in keeping them, we'd have to constantly remind people of them -- case in point -- which is a bit of a drag for everyone.

Btw, @principis, I agree that using specific pages for each subcommand is more useful than linking all of them to the homepage of the main command. Perhaps we should use a different term than "homepage" for the main link, to avoid giving out the wrong idea. WDYT?

**Comment by principis** [comment]
@waldyrious I added slashes consistently with the websites as was said in #2660 I find them ugly so I would rather remove them but it's not that big of a deal.

> Perhaps we should use a different term than "homepage" for the main link, to avoid giving out the wrong idea. WDYT?

Ow I forgot to mention that sorry :laughing: It was my intention to mention it in my PR.
I couldn't think of a good term but I think it's a good idea.

**Comment by jedahan** [comment]
maybe we don't specify and let people put whatever they think is most appropriate? Most likely it will be
`homepage: http://mycoolproject.org`
but sometimes
`documentation: http://project.readthedocs.org`
and maybe even
`community: http://project.gitter.im`

**Comment by sbrl** [comment]
@waldyrious: Oops! Missed that :P I'm unsure that it's worth ensuring that _all_ urls are formatted in _exactly_ the same way though.

@jedahan: Sounds like a great idea!

**Comment by sbrl** [comment]
Sounds good to me, @waldyrious

File: pages/common/youtube-dl.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # youtube-dl
 
 > Download videos from YouTube and other websites.
+> Homepage: <http://rg3.github.io/youtube-dl/>.
```
Comment (vladimyr) [inline pages/common/youtube-dl.md]: Prefer `https` if it is available :bowtie:

Comment (vladimyr) [reply pages/common/youtube-dl.md]: @sbrl Is this linter (https://github.com/tldr-pages/tldr-lint) still used for validating tldr pages? If so this should be added as rule there...

Comment (agnivade) [reply pages/common/youtube-dl.md]: Did you mean to comment on this PR ? What has tldr-lint got to do with adding a homepage link ?

Comment (vladimyr) [reply pages/common/youtube-dl.md]: In ideal world tldr-lint or any other linter used for checking tldr pages should report if http (home)page is linked and https version is available. It should report dead links also?

Comment (agnivade) [reply pages/common/youtube-dl.md]: Oh I see. I think that would put too much logic into the linter. I would want the heading text to be free-form without any strict format. The `Homepage:` format is just something we agreed to. I think it is not necessary to have linter checks for that.

Comment (vladimyr) [reply pages/common/youtube-dl.md]: Maybe I wasn't clear enough. I wasn't talking about homepage specifically but all urls page may contain. If url is found and uses less secure scheme or points to dead end that should be reported...

Comment (agnivade) [reply pages/common/youtube-dl.md]: I see. I am neutral to it.

Comment (sbrl) [reply pages/common/youtube-dl.md]: Some commands' homepages may be only available over http and not https though.

Comment (vladimyr) [reply pages/common/youtube-dl.md]: Quoting myself:
> if http (home)page is linked and https version is available.

That is relatively easy to check.

Comment (vladimyr) [reply pages/common/youtube-dl.md]: This is probably not a place to discuss it but parsing tldr pages with bison is like killing fly with orbital ion cannon 🙄
After all tldr page is markdown document and there are plenty of markdown parsers around that can be adapted/wrapped to check page format. Such parser could provide hooks for specific page segment like urls which can in turn be used for http/https and/or dead link checking.

**Comment by sbrl** [comment]
What's remaining for getting this and #2660 merged, @agnivade?

**Comment by agnivade** [comment]
We were waiting for the node client to have home pages support. Currently, the node client breaks trying to display the home page link. 

@vladimyr - You had mentioned about subclassing the markdown parser to add the support. Are you planning to do that ?

**Comment by vladimyr** [comment]
Not atm :disappointed: Right now I don't have enough free time for doing it properly including extensive test suite but maybe some time in the future...

**Comment by vladimyr** [comment]
Sure, totally agree 👍

**Comment by agnivade** [comment]
Done. Merging this.

File: pages/common/youtube-dl.md (hunk 2 above)
Comment (agnivade) [reply pages/common/youtube-dl.md]: Couldn't agree more. Sadly, that is the state of affairs now. If someone wants to rewrite the linter completely, they are more than welcome.

**Pull request merged**